### PR TITLE
Adding alert support

### PR DESCRIPTION
**Pull request by mauricioborges**
As said in issue #73, I'd like to have allert support on python-librato client.

I made a partial support actually according to my needs, but i'd like to have a feedback on it before keeping my approach. Some considerations:
* I did a sort of fluent interface to include the alerts
* I created a different test class in integration module
* I based strongly on instrument implementation

### CODE REVIEW

**Comment by jderrett** [comment]
@mauricioborges this looks pretty good so far...I'm going to test a few things here.  Meanwhile, could you squash these commits a bit?  Thanks!

**Comment by mauricioborges** [comment]
@jderrett I did that squash, looking forward for your recommendations if any. Thanks

**Comment by jderrett** [comment]
@mauricioborges I'm sorry for the delay on this...trying to get this in as soon as I can, which should be early this week.

**Comment by mauricioborges** [comment]
@jderrett thanks for the feedback. I'm aware that's a little big PR, I'll wait for your answer :-).

File: librato/alerts.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        self.duration = None
+        return self
+
+    def stops_reporting_for(self, duration, summary_function=None):
+        self.condition_type = 'absent'
+        self.summary_function = summary_function
+        self.duration = duration
+        return self
+
+    def during(self, duration):
```
Comment (jderrett) [inline librato/alerts.py]: Let's change this to `duration()` as in `below(42).duration(10)`

Comment (mauricioborges) [inline librato/alerts.py]: you mean stops_reporting_for().duration()?

Comment (jderrett) [inline librato/alerts.py]: More like https://github.com/mauricioborges/python-librato/blob/adding-alert-support/README.md#alerts where you've added the example `alert.add_condition_for('metric_name').above(1).during(60)`.  The threshold alerts can either be immediate, or have a duration, like "below 42 for 60 seconds".  So I'd prefer `alert.add_condition_for('metric_name').above(1).duration(60)`

Make sense?

**Comment by jderrett** [comment]
@mauricioborges also there's a small merge conflict in tests/integration.py.  Not sure if you want to merge with `master` on your end (or I can just merge your PR locally and push to master).  Thanks.

**Comment by mauricioborges** [comment]
@jderrett, i did a rebase a few hours ago to fix it. I'm on my way now,when I arrive at home I'll check and fix both things

**Comment by mauricioborges** [comment]
@jderrett I've reworked some commits, sorry for that. Now I've added all the integration tests (all of them running) and rebased from master.

File: librato/alerts.py (hunk 1 above)
Comment (mauricioborges) [inline librato/alerts.py]: Yes, makes more sense. Did this change on commit 0372d1a

**Comment by jderrett** [comment]
@mauricioborges merged, thanks again!  I'll make another pass over this and cut a new release shortly.

**Comment by jderrett** [comment]
Done in #76

**Comment by mauricioborges** [comment]
@jderrett thanks for all. My 2nd PR accepted in a OSS :D

**Comment by jderrett** [comment]
@mauricioborges great!  Thanks again for the contributions, these are super helpful.  I'll merge my open PR and then get this released today.

**Pull request merged**